samples/matter/thermostat/src/temperature_manager.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "app_event.h"
+
+#include <app-common/zap-generated/attributes/Accessors.h>
+
+#include <lib/core/CHIPError.h>
+
+using namespace chip;
+
+// AppCluster Spec Table 85.
+enum : uint8_t {
```

Comment:
I guess the enum like this one should be defined somewhere in the Matter stack / Matter clusters. Could you try to find it and maybe you will avoid a duplicate.

Reply:
Moreover, I've checked the spec and I can see more than 5 entries there. What about the others? :)